plugin/storage/es/options.go

```diff
@@ -221,6 +223,10 @@ func addFlags(flagSet *flag.FlagSet, nsConfig *namespaceConfig) {
 		nsConfig.namespace+suffixCreateIndexTemplate,
 		nsConfig.CreateIndexTemplates,
 		"Create index templates at application startup. Set to false when templates are installed manually.")
+	flagSet.Uint(
+		nsConfig.namespace+suffixVersion,
+		0,
+		"The major Elasticsearch version. The not specified value enables auto-detection from Elasticsearch API.")
```

Comment:
"The not specified value ..." -> "If not specified, the value will be auto-detected from Elasticsearch"?